build/bin.LICENSE

```diff
@@ -2207,3 +2207,537 @@
       THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
       (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
       THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+  This product bundles 'Saxon-HE (Home Edition)', including the following files:
```

Comment:
Actually we can remove the Saxon license because daffodil-debugger does not depend on daffodil-schematron, and therefore does not include Saxon or its transitive dependencies in the daffodil-debugger zip file.  It's important to build the extension and then look inside daffodil-debugger.zip to see which jars daffodil-debugger actually uses instead of simply starting with a copy of Daffodil's bin.LICENSE and bin.NOTICE.  I was surprised how many differences I found between Daffodil's libs and daffodil-debugger's libs once I actually compared the libs.  I don't simply see a superset or subset of one's libs in the other's libs; some dependencies even have different versions.  We only just released Daffodil 3.3.0, so there hasn't been enough time for Daffodil to bump some dependency versions to newer versions yet.  Someone will have to look closely at how daffodil-debugger specifies its dependencies and figure out why certain dependencies have different versions.

P.S.  I know why the versions are different now - when I built the extension, I'd checked out the main branch but the main branch is still using Daffodil 3.2.1, not 3.3.0.